resources/run_adl.sh

```diff
@@ -184,6 +190,7 @@ if [[ $? -ne 0 ]]; then
     export LIBRARY_PATH="$(pwd)/artifacts"
 
     go mod download
+    go mod vendor
```

Comment:
The vendor step isn't needed here.  Check in your environment if you have `GOFLAGS="-mod=vendor"` in your environment.  If you unset that then the script will work without the `go mod vendor` line.

Reply:
oh yes, that's right.
Removed :)